app/views/admin/signs/_collection.html.erb

```diff
@@ -54,7 +54,7 @@ to display a collection of resources in an HTML table.
           >
         <% collection_presenter.attributes_for(resource).each do |attribute| %>
           <td class="cell-data cell-data--<%= attribute.html_class %>">
-            <% if show_action? :show, resource -%>
+            <% if authorized_action?(resource, :show) -%>
```

Comment:
note: we're using `authorized_action?` here instead of `accessible_action?` like we do in other places

In this case `accessible_action?` returns false due to checking `existing_action?` first (which checks the routes), which I believe returns false because the particular route for this is not an admin route - I assume there probably is something we could do like define a route alias and that might even be the ideal thing to do, but I've not done it since it would be more complex and might have other tradeoffs and meanwhile `authorized_action?` is still valid 🤷